### PR TITLE
components: (google-cloud-sdk) Add supported OP names and description to readme.

### PR DESCRIPTION
**Pull request by SinaChavoshi**
Adding name and description of supported OPs to readme file.

### CODE REVIEW

File: components/google-cloud/README.md [hunk 1]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    ### Supported Components 
+    Following is the list of currently supproted components. 
+
+    ```python
+    AutoMLImageTrainingJobRunOp(...)
+        """Runs the AutoML Image training job and returns a model."""
+
+    AutoMLTabularTrainingJobRunOp(...)
+        """Runs the training job and returns a model."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:92]:
```suggestion
        """Runs an AutoML Tabular training job and returns a model."""
```

File: components/google-cloud/README.md [hunk 2]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
\ No newline at end of file
+    ```
+
+
+    ### Supported Components 
+    Following is the list of currently supproted components. 
+
+    ```python
+    AutoMLImageTrainingJobRunOp(...)
+        """Runs the AutoML Image training job and returns a model."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:89]:
```suggestion
        """Runs an AutoML Image training job and returns a model."""
```

File: components/google-cloud/README.md [hunk 3]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    AutoMLTabularTrainingJobRunOp(...)
+        """Runs the training job and returns a model."""
+
+    AutoMLTextTrainingJobRunOp(...)
+        """Runs the training job and returns a model."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:95]:
```suggestion
        """Runs an AutoML Text training job and returns a model."""
```

File: components/google-cloud/README.md [hunk 4]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    AutoMLVideoTrainingJobRunOp(...)
+        """Runs the AutoML Image training job and returns a model."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:98]:
```suggestion
        """Runs an AutoML Image training job and returns a model."""
```

File: components/google-cloud/README.md [hunk 5]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    CustomContainerTrainingJobRunOp(...)
+        """Runs the custom training job."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:101]:
```suggestion
        """Runs a custom container training job."""
```

File: components/google-cloud/README.md [hunk 6]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    CustomPythonPackageTrainingJobRunOp(...)
+        """Runs the custom training job."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:104]:
```suggestion
        """Runs a custom Python package training job."""
```

File: components/google-cloud/README.md [hunk 7]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    EndpointCreateOp(...)
+        """Creates a new endpoint."""
+
+    ImageDatasetCreateOp(...)
+        """Creates a new image dataset and optionally imports data into dataset when"""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:110]:
```suggestion
        """Creates a new image dataset and optionally imports data into the dataset."""
```

File: components/google-cloud/README.md [hunk 8]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    ImageDatasetExportDataOp(...)
+        """Exports data to output dir to GCS."""
+
+    ImageDatasetImportDataOp(...)
+        """Upload data to existing managed dataset."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:116]:
```suggestion
        """Upload Image data to existing managed Image dataset."""
```

File: components/google-cloud/README.md [hunk 9]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    ModelBatchPredictOp(...)
+        """Creates a batch prediction job using this Model and outputs prediction"""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:119]:
```suggestion
        """Creates a batch prediction job using this Model and outputs prediction."""
```

File: components/google-cloud/README.md [hunk 10]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+        """Deploys model to endpoint. Endpoint will be created if unspecified."""
+
+    ModelUploadOp(...)
+        """Uploads a model and returns a Model representing the uploaded Model resource."""
+
+    TabularDatasetCreateOp(...)
+        """Creates a new tabular dataset."""
+
+    TabularDatasetExportDataOp(...)
+        """Exports data to output dir to GCS."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:131]:
```suggestion
        """Exports Tabular data to output dir to GCS."""
```

File: components/google-cloud/README.md [hunk 11]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    TabularDatasetImportDataOp(...)
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:133]: This should be removed. Tabular dataset do not support import. I'll remove the op in a follow up PR.

File: components/google-cloud/README.md [hunk 12]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+        """Upload data to existing managed dataset."""
+
+    TextDatasetCreateOp(...)
+        """Creates a new text dataset and optionally imports data into dataset when"""
+
+    TextDatasetExportDataOp(...)
+        """Exports data to output dir to GCS."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:140]:
```suggestion
        """Exports Text data to output dir to GCS."""
```

File: components/google-cloud/README.md [hunk 13]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    TextDatasetImportDataOp(...)
+        """Upload data to existing managed dataset."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:143]:
```suggestion
        """Upload data to existing managed Text dataset."""
```

File: components/google-cloud/README.md [hunk 14]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    VideoDatasetCreateOp(...)
+        """Creates a new video dataset and optionally imports data into dataset when"""
+
+    VideoDatasetExportDataOp(...)
+        """Exports data to output dir to GCS."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:149]:
```suggestion
        """Exports Video data to output dir to GCS."""
```

File: components/google-cloud/README.md [hunk 15]
```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
...
+
+    VideoDatasetImportDataOp(...)
+        """Upload data to existing managed dataset."""
```
Comment (sasha-gitg) [inline components/google-cloud/README.md:152]:
```suggestion
        """Upload data to existing managed Video dataset."""
```

**Review by sasha-gitg — APPROVED** [review]

File: components/google-cloud/README.md (hunk 11 above)
Comment (SinaChavoshi) [reply components/google-cloud/README.md]: got it, I removed the note from read me and the op from Init.

**Pull request merged**